renovate.json

```diff
@@ -1,46 +1,87 @@
+
 {
- "extends": ["config:base"],
- "baseBranches": ["master"],
- "bumpVersion": null,
- "packageRules": [
-   {
-     "depTypeList": ["devDependencies"],
-     "updateTypes": ["minor"],
-     "automerge": false
-   }
- ],
- "prHourlyLimit": 5,
- "prConcurrentLimit": 5,
- "rangeStrategy": "bump",
- "renovateFork": true,
- "includeForks": true,
- "regexManagers": [
+  "extends":[
+    "config:base"
+  ],
+  "enabled": "true",
+  "baseBranches": ["master"],
+  "timezone": "Europe/Helsinki",
+  "packageRules": [
     {
-      "fileMatch": ["^lib/common.sh$"],
-      "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
-      "depNameTemplate": "kubernetes/minikube",
-      "datasourceTemplate": "github-releases"
+      "depTypeList": ["devDependencies"],
+      "matchUpdateTypes": ["minor"],
+      "automerge": true
     },
     {
-      "fileMatch": ["^lib/common.sh$"],
-      "matchStrings": ["KIND_VERSION:-\"(?<currentValue>.*?)\"}"],
-      "depNameTemplate": "kubernetes-sigs/kind",
-      "datasourceTemplate": "github-releases"
+      "datasources": ["docker"],
+      "packageNames": ["kindest/node"],
+      "automerge": true 
+    },    
+    {
+      "datasources": ["pypi"],
+      "packageNames": ["ansible"],
+      "automerge": true 
     }
+
   ],
-  "assignees": ["fmuyassarov", "kashifest"],
-  "assigneesSampleSize": 1,
-  "reviewers": [
-      "furkatgofurov7",
-      "jan-est",
-      "lentzi90",
-      "macaptain",
-      "mboukhalfa",
-      "namnx228",
-      "smoshiur1237",
-      "Sunnatillo",
-      "wgslr",
-      "Xenwar"
-  ],
-  "reviewersSampleSize": 2
-}
+  "prHourlyLimit": 5,
+  "prConcurrentLimit": 5,
+  "rangeStrategy": "bump",
+  "renovateFork": true,
+  "includeForks": false,
+  "stabilityDays":"5",
+  "regexManagers": [
+     {
+       "fileMatch": ["^lib/common.sh$"],
+       "matchStrings": ["KUBERNETES_VERSION:-\"(?<currentValue>.*?)\"}"],
+       "depNameTemplate": "kubernetes/kubernetes",
+       "datasourceTemplate": "github-releases"
+     },
+     {
+       "fileMatch": ["^lib/common.sh$"],
+       "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
+       "depNameTemplate": "kubernetes/minikube",
+       "datasourceTemplate": "github-releases"
+     },
+     {
+       "fileMatch": ["^lib/common.sh$"],
+       "matchStrings": ["KIND_VERSION:-\"(?<currentValue>.*?)\"}"],
+       "depNameTemplate": "kubernetes-sigs/kind",
+       "datasourceTemplate": "github-releases"
+     },
+     {
+       "fileMatch": ["^lib/common.sh$"],
+       "matchStrings": ["KUSTOMIZE_VERSION:-\"(?<currentValue>.*?)\"}"],
+       "depNameTemplate": "kubernetes-sigs/kustomize",
+       "datasourceTemplate": "github-releases"
```

Comment:
Will this work with kustomize's "weird" release tags? I have had some trouble with them previously so just want to confirm.
The issue I have bumped into is that they release multiple different packages independently of each other. This means that there can be a `kustomize/v4.2.0` one day and the next there is `kyaml/v0.11.0`. So my question is: is renovate smart enough to just grab the `kustomize/*` tags?

Reply:
Thanks, I realized that it is not grubbing customize tags. To solve this issue we have to change other parts of the code. So I will just skip kustomize for now